### ISSUE TITLE
different SFR

### ISSUE THREAD

**Issue by Amarchuk** [LerkaKostuik/Course-work#15]
Мы хотели сравнить еще разные индикаторы SFR между собой. Вот что я предлагаю:

* формула (D11) из https://arxiv.org/pdf/0810.2556.pdf - FUV+24mm. Они обе есть в NED - искать по Ctrl+F SINGS и выбрать 24um, и вторая - GALEX FUV (по идее вторую еще надо исправлять за поглощение, но пока можно без - см 2 в https://arxiv.org/pdf/1507.03801.pdf)
* Halpha+24mm отсюда https://arxiv.org/pdf/0705.3377.pdf (7)
* или отсюда https://arxiv.org/pdf/1301.2328.pdf те же индикаторы (формула 7)
* или любой из 3х ИК отсюда https://arxiv.org/pdf/1606.08845.pdf

Идея следующая - не нужно сравнивать все, а просто сделать две карты SFR из разных индикаторов (одна только из Halpha у нас уже есть, так что можно и всего одну) и показать, что Sigma_SFR у двух разных индикаторов хорошо совпадает. Это убедит нас, что все верно, и рецензента.  Если не совпадает - то будем думать (и искать две, что похожи)

Еще полезные ссылки - https://arxiv.org/pdf/1507.03801.pdf (формулы) и глава 3.1 в https://arxiv.org/pdf/1212.4152.pdf (это собственно та же группа, чьи данные мы использовали - там есть две карты SFR для NGC628, можете посмотреть).

**Comment by ValerieKostiuk** [comment]
Я создала отдельный ноутбук, в котором мы будем работать над этой задачей.
У меня появилась сейчас на начальном этапе небольшая проблема. Я с ней уже долго мучилась и решила попросить Вашей помощи.
В новом ноутбуке я сейчас пыталась обрезать данные I_24 и I_FUV по данным H_alpha. И в обоих случаях у меня появлялась ошибка: `NoOverlapError: Arrays do not overlap`. Я прочитала на сайте разработчиков astropy об этой ошибке и мне удалось обрезать данные I_24, но с I_FUV никак не получается. Там нужно поменять параметр mode у Cutout2D, но как бы я не меняла, ничего не исправляется. Может Вы увидите, где спрятана ошибка.

**Comment by Amarchuk** [comment]
И правильно сделали. Как мне кажется, там либо с координатами что-то не то в самом фитсе, либо массив не двухмерный. 
Попробуйте считать фитс и посмотреть на `shape` его данных или вот этим советом https://stackoverflow.com/questions/55207456/astropy-cutout2d-raise-valueerror (последний комментарий и единственный ответ). Если не выйдет - я попробую сам.

**Comment by ValerieKostiuk** [comment]
Я посмотрела массив у меня всё-таки двумерный. И правда возможно проблемы с координатами. Попробую ещё над этим вопросом посидеть.

**Comment by Amarchuk** [comment]
Хорошо. Если больше двух часов не получается - лучше попробовать что-то другое (например другой фитс скачать, оригинал поискать и т.д.).

**Comment by ValerieKostiuk** [comment]
У меня получилось обрезать эти данные, но я не уверена, что это сделалось очень хорошо. Проблема мы была в том, что в Header этого fits файла видимо есть ошибка. Значения координат центра галактики таковы, что они соответствуют отрицательным значениям пикселей, т.е. центр галактики не будет находиться на наших данных. Так что обрезать мне пришлось по значениям координат центра данных из Venga. Ничего же страшного в этом нет? В принципе значения координат центра должны быть одинаковыми для всех данных.

**Comment by Amarchuk** [comment]
Понятно, как мы и думали, проблема с заголовками. Да, конечно обрезать так вполне нормально - я посмотрел картинки, они вполне достойные.

**Comment by ValerieKostiuk** [comment]
Хорошо, тогда я могу продолжать работу с этим ноутбуком. Вот я только одно сейчас не могу понять в этих фитсах же даны интенсивности  FUV и 24mm? А единицы измерения этих величин  MJy ster−1? Я просто в статьях не могу это найти, может это очевидный факт.

**Comment by Amarchuk** [comment]
Нет, не очевидный и я сходу не знаю. Тут нужно либо идти в статью (самая правая колонка на NED) и искать подробности, либо применять формулу и если получились адекватные цифры - то все хорошо. Еще можно попробовать сайт проекта/обсерватории найти и там фитсы посмотреть, сравнить.
